test/functional/wallet_listtransactions.py

```diff
@@ -234,8 +234,8 @@ def run_externally_generated_address_test(self):
         # refill keypool otherwise the second node wouldn't recognize addresses generated on the first nodes
         self.nodes[0].keypoolrefill(1000)
         self.stop_nodes()
-        wallet0 = os.path.join(self.nodes[0].chain_path, self.default_wallet_name, "wallet.dat")
```

Comment:
In commit "Fix wallet directory initialization" (17b4955e4b1e4f02f00a948b432f18caa9b250a5)

Is there a reason this test is changing? I'd expect the change to be backwards compatible so existing tests should not need to change

Reply:
@ryanofsky This is because of the change in wallet dir init logic. The `wallet/` dir is created when a new node is started and no wallet is detected in data_dir